From akpm@osdl.org Wed Nov 30 14:00:03 2005
Message-Id: <200511302156.jAULuQ4M011629@shell0.pdx.osdl.net>
From: Paul Walmsley <paul@booyaka.com>
Subject: usb-storage: add debug entry for REPORT LUNS
Subject: + usb-storage-add-debug-entry-for-report-luns.patch added to -mm tree
To: paul@booyaka.com, greg@kroah.com, mdharm-usb@one-eyed-alien.net
Date: Wed, 30 Nov 2005 13:57:45 -0800

From: Paul Walmsley <paul@booyaka.com>

Bugs involving the REPORT LUNS SCSI-3 command are much easier to track
down if usb-storage displays the command's name, rather than "(Unknown
command)".

Signed-off-by: Paul Walmsley <paul@booyaka.com>
Cc: <mdharm-usb@one-eyed-alien.net>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/usb/storage/debug.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/storage/debug.c
+++ gregkh-2.6/drivers/usb/storage/debug.c
@@ -132,6 +132,7 @@ void usb_stor_show_command(struct scsi_c
 	case 0x5C: what = "READ BUFFER CAPACITY"; break;
 	case 0x5D: what = "SEND CUE SHEET"; break;
 	case GPCMD_BLANK: what = "BLANK"; break;
+	case REPORT_LUNS: what = "REPORT LUNS"; break;
 	case MOVE_MEDIUM: what = "MOVE_MEDIUM or PLAY AUDIO (12)"; break;
 	case READ_12: what = "READ_12"; break;
 	case WRITE_12: what = "WRITE_12"; break;
